src/sentry/web/helpers.py

```diff
@@ -20,7 +20,8 @@ def get_default_context(request, existing_context=None, team=None):
         "URL_PREFIX": options.get("system.url-prefix"),
         "SINGLE_ORGANIZATION": settings.SENTRY_SINGLE_ORGANIZATION,
         "PLUGINS": plugins,
-        "ONPREMISE": settings.SENTRY_ONPREMISE,
+        "ONPREMISE": settings.SENTRY_SELF_HOSTED,  # deprecated, use ...
+        "SELF_HOSTED": settings.SENTRY_SELF_HOSTED,  # ... this instead
```

Comment:
What are these contexts? What consumes them?

Reply:
Introduced in #3350 so Django templates can vary:

https://github.com/getsentry/sentry/blob/e99df7ffb42ee8640e631520e2997a4b387e2e62/src/sentry/templates/sentry/layout.html#L193

We're not using this anymore, but if we remove this then we should probably remove `SINGLE_ORGANIZATION` as well, but I want to be conservative here so I am leaving these in.